packages/console/src/components/Sidebar/components/Item/index.tsx

```diff
@@ -0,0 +1,21 @@
+import classNames from 'classnames';
+import React, { ReactChild } from 'react';
+
+import * as styles from './index.module.scss';
+
+type Props = {
+  icon?: ReactChild;
+  title: string;
+  isActive?: boolean;
+};
+
+const Item = ({ icon, title, isActive = false }: Props) => {
+  return (
+    <div className={classNames(styles.row, isActive && styles.active)}>
+      {icon && <div className={styles.icon}>{icon}</div>}
+      <div className={styles.title}>{title}</div>
+    </div>
+  );
+};
```

Comment:
Simplify return by omitting the outer parenthesis? This could've be in the lint rules, I think?

Reply:
Yeah, that's true. I feel you, I really do. 😂 Then we leave it as-is for now

Reply:
cool